### ISSUE TITLE
Dropdown menu on mobile device doesn't open.

### ISSUE THREAD

**Issue by kaganvmz** [kybarg/bootstrap-dropdown-hover#7]
Second-level menu or more doesn't open, on mobile devices.
![screenshot_4](https://cloud.githubusercontent.com/assets/7592740/21133440/7b7573da-c121-11e6-98be-60060a33ca5b.png)

**Comment by anlek** [comment]
@kaganvmz have you found a work around?

**Comment by promojit** [comment]
For the mobile devices, there is a problem for the multi-level dropdown. The solution I found is this -

In **html** 
```
<li class="dropdown">
    <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false" data-hover="dropdown" data-animations="flipInX flipInX flipInX flipInX">Dropdown<span class="caret"></span></a>
    <ul class="dropdown-menu">
        <li><a href="#">Option 1</a></li>
        <li><a href="#">Option 2</a></li>
        <li class="dropdown dropdown-submenu">
            <a class="test" tabindex="-1"  href="#">Option 3 <span class="caret"></span></a>
            <ul class="dropdown-menu">
              <li><a href="#">Option 3.1</a></li>
              <li><a href="#">Option 3.2</a></li>
              <li><a href="#">Option 3.3</a></li>
              <li><a href="#">Option 3.4</a></li>

            </ul>
        </li>
        <li><a href="#">Option 4</a></li>
        <li><a href="#">Option 5</a></li>
        <li class="dropdown dropdown-submenu">
            <a class="test" tabindex="-1"  href="#">Option 6 <span class="caret"></span></a>
            <ul class="dropdown-menu">
              <li><a href="#">Option 6.1 </a></li>
              <li><a href="#">Option 6.2</a></li>
              <li><a href="#">Option 6.3</a></li>
            </ul>
        </li>
    </ul>
</li> 
```
And then add this **js**
```
$(window).on('resize', function(event){
        var windowSize = $(window).width();
        if(windowSize < 768){
            $('.dropdown-submenu .test').on("click", function(e){
                $(this).next('ul').toggle();
                e.stopPropagation();
                e.preventDefault();
            });
        }
    });
```
It works for me. Wish it will help.

**Comment by kybarg** [comment]
Fixed in 1.1.0 version
